### PR TITLE
evm: update readme.MD

### PR DESCRIPTION
**Pull request by panoel**

### CODE REVIEW

File: .cspell/custom-dictionary.txt [hunk 1]
```diff
@@ -16,3 +16,4 @@ struct
 thresholding
 trustlessness
 upgradability
+tooManytransceivers
```
Comment (evan-gray) [inline .cspell/custom-dictionary.txt:19]: Shouldn't the `t` in Transceivers be capitalized? then it probably wouldn't have had a spellcheck error.

File: evm/README.md [hunk 2]
```diff
@@ -1,4 +1,207 @@
-## Foundry
+# EVM Documentation
+
+## Design
+
+An Integrator is the on-chain contract which will call sendMessage on the Router. From the Router’s perspective, the Integrator is the msg.sender of sendMessage. Before an Integrator is able to send their first message, they MUST register with the Router in order to set their admin, and that admin then MUST subsequently register at least one Transceiver and enable it for sending on at least one chain. The Router MUST enforce that admin functions specify the Integrator address as a parameter and revert when msg.sender is NOT the provided Integrator’s admin.
```
Comment (evan-gray) [inline evm/README.md:5]: please use backticks for the code references, like `msg.sender` and `sendMessage`?

File: evm/README.md [hunk 3]
```diff
@@ -1,4 +1,207 @@
...
+
+Integrator ->> Router: register(admin)
+note over Router: The AdminContract is now the only admin of the integrator
+note over Router:emit IntegratorRegistered(integrator, admin)
+AdminContract->>Router:addTransceiver(integrator, chain, address)
+alt transceiver already registered
+Router-->>AdminContract:revert TransceiverAlreadyRegistered
+end
+alt numTransceivers >= MAX_TRANSCEIVERS
+Router-->>AdminContract:revert tooManytransceivers
```
Comment (evan-gray) [inline evm/README.md:27]: `tooManyTransceivers` ?

File: evm/README.md [hunk 4]
```diff
@@ -1,4 +1,207 @@
...
+note over AdminContract, Router:Cancels transfer
+note over Router:emit AdminUpdated(integrator, admin, admin)
+note over Router:or
+NewAdmin->>Router:claimAdmin(integrator)
+note over Router, NewAdmin:Second step to accept the transfer
+note over Router:emit AdminUpdated(integrator, admin, newAdmin)
+note over Router:is acceptable
+```
+
+### Message Flow:
```
Comment (evan-gray) [inline evm/README.md:77]: I feel like this section can be removed for now and potentially replaced with diagrams like the above in the future. The diagrams definitely seem more helpful than the pseudo code.

**Review by evan-gray — CHANGES_REQUESTED** [review]
few small things

File: evm/README.md (hunk 2 above)
Comment (panoel) [reply evm/README.md:5]: Added

File: evm/README.md (hunk 3 above)
Comment (panoel) [reply evm/README.md:27]: Changed

File: evm/README.md (hunk 4 above)
Comment (panoel) [reply evm/README.md:77]: Removed

File: .cspell/custom-dictionary.txt (hunk 1 above)
Comment (panoel) [reply .cspell/custom-dictionary.txt:19]: Removed

Comment (panoel) [reply .cspell/custom-dictionary.txt]: Removed

**Review by evan-gray — APPROVED** [review]

**Pull request merged**